### ISSUE TITLE
PGCRs: data on reward tier (bronze/silver/gold/plat)

### ISSUE THREAD

**Issue by d2orbc** [Bungie-net/api#1841]
Would be great if we had info on what tier the nightfall ended on.

Particularly needed for GM leaderboards, where it would be nice to see if a fast run was plat or not.

**Comment by justrealmilk** [comment]
✅

**Comment by d2orbc** [comment]
Yes, still a valid request. Vanguard medals and in-game timer too.
